developer_manual/core/unit-testing.rst

```diff
@@ -253,3 +253,49 @@ Here are some useful links about how to write unit tests with Jasmine and Sinon:
    
 .. _the PHPUnit documentation: https://phpunit.de/manual/current/en/installation.html
 .. _the writing tests section: http://www.phpunit.de/manual/current/en/writing-tests-for-phpunit.html
+
+
+UI testing in Core with selenium
+--------------------------------
+
+Requirements:
+~~~~~~~~~~~~~
+- ownCloud >= 10.0 (make sure you have an running instance of ownCloud complete setup)
+- admin user called "admin" with the password "admin"
+- no self-signed SSL certificates
+- testing utils (running ``make`` in your terminal from the ``webroot`` directory will install them)
+- Selenium standalone server (download Selenium standalone server jar from http://docs.seleniumhq.org/download/)
+- browser installed that you would like to test on
+- webdriver for the browsers you want to test (e.g. download the chromedriver from: https://sites.google.com/a/chromium.org/chromedriver/ , firefox webdriver is included in the selenium server)
+
+Set Up test
+~~~~~~~~~~~
+- place the selenium standalone server jar file and the webdriver(s) somewhere in the same folder
+- start the selenium server ``java -jar selenium-server-standalone-3.0.1.jar -port 4445``
+- set the following environment variables:
+  - SRV_HOST_NAME (the hostname where ownCloud runs)
+  - SRV_HOST_URL (path if ownCloud does not run in the root of the host)
+  - SRV_HOST_PORT (port of your webserver)
+  - BROWSER (chrome, firefox, internet explorer)
+
+  e.g. to test an instance running on http://localhost/owncloud-core with chrome do:
+
+  ::
+
+    export SRV_HOST_NAME=localhost
+    export SRV_HOST_URL=owncloud-core
+    export SRV_HOST_PORT=80
+    export BROWSER=chrome
+
+
+- if you don't have a webserver already running start php development server with:
+  ``bash tests/travis/start_php_dev_server.sh`` (leave SRV_HOST_URL empty in that case. ``export SRV_HOST_URL=""``)
+  The server will bind to: $SRV_HOST_NAME:$SRV_HOST_PORT
```

Comment:
I'd suggest that you surround $SRV_HOST_NAME:$SRV_HOST_PORT with double backticks, to render it as code.